index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css">
+    <title>Hello Microverse</title>
+  </head>
+  <body>
+    <h1>Hello Microverse</h1>
```

Comment:
- Nicely done, I was able to see your first microverse application in the browser:🤗